__tests__/app.test.js

```diff
@@ -26,24 +27,59 @@ describe("GET /api/topics", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
-      .then(({ body}) => {
-        console.log({body});
-        expect(body.topics).toHaveProperty("GET /api/topics");
-        body.topics["GET /api/topics"].exampleResponse.topics.forEach(topic => {
+      .then(({ body: {topics}}) => {
+        topics.forEach(topic => {
           expect(topic).toHaveProperty("slug");
           expect(topic).toHaveProperty("description")
         });
 
       });
+  }); 
+});
+
+describe("All bad urls", () => {
+  test("404: Responds with error when route does not exist", () => {
+  return request(app)
+    .get("/api/nonexistent")
+    .expect(404)
+    .then(({ body: {message} }) => {
+     expect(message).toBe("Not Found");
+    });
   });
-  test.only("404: Responds with error when route does not exist", () => {
+})
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object based on the given article_id", () => {
     return request(app)
-      .get("/api/nonexistent")
-      .expect(404)
-      .then(({ body }) => {
-        console.log({body});
-        
-        expect(body.message).toBe("Not Found");
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body: {article}}) => {
+        expect(article).toHaveProperty("author", "butter_bridge");
```

Comment:
Ha. I had no idea the toHaveProperty matcher takes a second optional argument of a value - very cool!